projects/cashmere/src/lib/date-range/picker-overlay/picker-overlay.component.scss

```diff
@@ -0,0 +1,41 @@
+.hc-date-range-calendar-container {
+    display: flex;
+    justify-content: space-around;
+    min-width: 650px;
+}
+
+.hc-date-range-calendar-item {
+    flex-basis: 33.33%;
+    min-width: 210px;
+    padding: 1em;
+    font-family: Roboto, 'Helvetica Neue', sans-serif;
```

Comment:
Just noticed this one.  Should be open sans, not roboto - that's Angular Material showing through :-)

Reply:
Wow! good catch